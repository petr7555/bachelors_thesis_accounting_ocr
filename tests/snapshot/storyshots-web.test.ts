import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  // from project root
  configPath: './.storybook',
});
