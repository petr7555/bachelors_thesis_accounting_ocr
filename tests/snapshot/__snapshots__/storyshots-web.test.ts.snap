// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Buttons/PrimaryButton With Emoji 1`] = `
<View
  style={
    Array [
      Object {
        "borderRadius": 3,
        "overflow": "hidden",
      },
      Object {
        "borderRadius": 3,
      },
      Object {
        "borderRadius": 25,
      },
      false,
    ]
  }
>
  <View
    accessibilityRole="button"
    accessibilityState={
      Object {
        "busy": false,
        "disabled": false,
      }
    }
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#310783",
          "borderColor": "#2089dc",
          "borderRadius": 25,
          "borderWidth": 0,
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "center",
          "padding": 8,
          "width": 690,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 18,
            "paddingVertical": 1,
            "textAlign": "center",
          }
        }
      >
        😀 😎 👍 💯 🚀
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Buttons/PrimaryButton With Text 1`] = `
<View
  style={
    Array [
      Object {
        "borderRadius": 3,
        "overflow": "hidden",
      },
      Object {
        "borderRadius": 3,
      },
      Object {
        "borderRadius": 25,
      },
      false,
    ]
  }
>
  <View
    accessibilityRole="button"
    accessibilityState={
      Object {
        "busy": false,
        "disabled": false,
      }
    }
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#310783",
          "borderColor": "#2089dc",
          "borderRadius": 25,
          "borderWidth": 0,
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "center",
          "padding": 8,
          "width": 690,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 18,
            "paddingVertical": 1,
            "textAlign": "center",
          }
        }
      >
        Hello Button
      </Text>
    </View>
  </View>
</View>
`;
